apps/web/app/(use-page-wrapper)/settings/(settings-layout)/organizations/attributes/page.tsx

```diff
@@ -14,10 +17,24 @@ export const generateMetadata = async () =>
 
 const Page = async () => {
   const t = await getTranslate();
+  const routerCaller = await createRouterCaller(appRouter);
+  const user = await routerCaller.viewer.me.get();
```

Comment:
Let’s call get server session instead of hitting the me end point - this fetches way more data than we need